Formula/virt-manager.rb

```diff
@@ -56,41 +53,30 @@ class VirtManager < Formula
   end
 
   resource "libvirt-python" do
-    url "https://files.pythonhosted.org/packages/2a/74/919462bed158ccc2a8061f31d82c157740c2e597ff50490526336e8aa688/libvirt-python-8.1.0.tar.gz"
-    sha256 "a21ecfab6d29ac1bdd1bfd4aa3ef58447f9f70919aefecd03774613f65914e43"
+    url "https://files.pythonhosted.org/packages/02/13/1719711d9da5376df239ef185e1bc4aae2c59ae112a2eb6cf7ca612d1a3a/libvirt-python-8.6.0.tar.gz"
+    sha256 "81f49a648a4f3fbebf4abf3f8d4b1468654689d4df6fd21a303d1c1ca9344871"
   end
 
   resource "requests" do
-    url "https://files.pythonhosted.org/packages/60/f3/26ff3767f099b73e0efa138a9998da67890793bfa475d8278f84a30fec77/requests-2.27.1.tar.gz"
-    sha256 "68d7c56fd5a8999887728ef304a6d12edc7be74f1cfa47714fc8b414525c9a61"
+    url "https://files.pythonhosted.org/packages/a5/61/a867851fd5ab77277495a8709ddda0861b28163c4613b011bc00228cc724/requests-2.28.1.tar.gz"
+    sha256 "7c5599b102feddaa661c826c56ab4fee28bfd17f5abca1ebbe3e7f19d7c97983"
   end
 
   resource "urllib3" do
-    url "https://files.pythonhosted.org/packages/1b/a5/4eab74853625505725cefdf168f48661b2cd04e7843ab836f3f63abf81da/urllib3-1.26.9.tar.gz"
-    sha256 "aabaf16477806a5e1dd19aa41f8c2b7950dd3c746362d7e3223dbe6de6ac448e"
+    url "https://files.pythonhosted.org/packages/6d/d5/e8258b334c9eb8eb78e31be92ea0d5da83ddd9385dc967dd92737604d239/urllib3-1.26.11.tar.gz"
+    sha256 "ea6e8fb210b19d950fab93b60c9009226c63a28808bc8386e05301e25883ac0a"
   end
 
   def install
-    venv = virtualenv_create(libexec, "python3")
+    python = "python3.10"
+    venv = virtualenv_create(libexec, python)
     venv.pip_install resources
 
-    # virt-manager uses distutils, doesn't like --single-version-externally-managed
-    system libexec/"bin/python", "setup.py",
-                   "configure",
-                   "--prefix=#{libexec}"
-    system libexec/"bin/python", "setup.py",
-                     "--no-user-cfg",
-                     "--no-update-icon-cache",
-                     "--no-compile-schemas",
-                     "install"
-
-    # install virt-manager commands with PATH set to Python virtualenv environment
-    bin.install libexec.glob("bin/virt-*")
-    bin.env_script_all_files(libexec/"bin", PATH: "#{libexec}/bin:$PATH")
-
-    share.install libexec/"share/man"
-    share.install libexec/"share/glib-2.0"
-    share.install libexec/"share/icons"
+    args = Language::Python.setup_install_args(prefix, python)
+    args.insert((args.index "install"), "--no-update-icon-cache", "--no-compile-schemas")
+
+    system libexec/"bin/python", "setup.py", "configure", "--prefix=#{prefix}"
+    system libexec/"bin/python", *args
```

Comment:
Changes:
- Didn't see the issue with `--single-version-externally-managed` from previous comment, so just went with `setup_install_args`
- Just installed into `prefix` rather than `libexec`. Didn't see any reason to use `libexec` and seemed simpler than moving files afterward.
- Env scripts shouldn't be necessary. Setuptools `build_scripts` does shebang rewrite using running Python. Since venv python is used to run command, it should be set as shebangs and have access to venv-installed resources.
    ```console
    ❯ file /usr/local/opt/virt-manager/bin/*
    /usr/local/opt/virt-manager/bin/virt-clone:   a /usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python script text executable, ASCII text
    /usr/local/opt/virt-manager/bin/virt-install: a /usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python script text executable, ASCII text
    /usr/local/opt/virt-manager/bin/virt-manager: a /usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python script text executable, ASCII text
    /usr/local/opt/virt-manager/bin/virt-xml:     a /usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python script text executable, ASCII text
    
    ❯ ls /usr/local/opt/virt-manager/bin/* | xargs head -1
    ==> /usr/local/opt/virt-manager/bin/virt-clone <==
    #!/usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python
    
    ==> /usr/local/opt/virt-manager/bin/virt-install <==
    #!/usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python
    
    ==> /usr/local/opt/virt-manager/bin/virt-manager <==
    #!/usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python
    
    ==> /usr/local/opt/virt-manager/bin/virt-xml <==
    #!/usr/local/Cellar/virt-manager/4.1.0/libexec/bin/python
    ```